types/forge-viewer/index.d.ts

```diff
@@ -314,25 +315,41 @@ declare namespace Autodesk {
         }
 
         interface AggregatedViewInitOptions {
+          cameraValidator?: boolean;
+          clusterfck?: any;
```

Comment:
Wow, that really is a config property, isn't it 🤣

Reply:
Yep, it can be used for DI of clusterfck library which is used for clustering of bookmark icons.